python/myutils/kinFitterXbb.py

```diff
@@ -289,15 +310,27 @@ def processEvent(self, tree):
                     # higgs jet output vectors
                     fit_result_j1 = fitter.get4Vec(0)
                     fit_result_j2 = fitter.get4Vec(1)
-                    fit_result_H = fit_result_j1 + fit_result_j2
+                    fit_result_H_noJetMass = fit_result_j1 + fit_result_j2
+                    for iFsr in range(len(fit_jets)-2):
+                        fit_result_H_noJetMass += fitter.get4Vec(2+iFsr)
 
-                    fit_result_l1 = fitter.get4Vec(2)
-                    fit_result_l2 = fitter.get4Vec(3)
+                    # add back jet masses
+                    fit_result_j1.SetPtEtaPhiM(fit_result_j1.Pt(),fit_result_j1.Eta(),fit_result_j1.Phi(), mass[hJidx[0]] * tree.Jet_PtReg[hJidx[0]]/tree.Jet_Pt[hJidx[0]])
+                    fit_result_j2.SetPtEtaPhiM(fit_result_j2.Pt(),fit_result_j2.Eta(),fit_result_j2.Phi(), mass[hJidx[1]] * tree.Jet_PtReg[hJidx[1]]/tree.Jet_Pt[hJidx[1]])
```

Comment:
Just wondering here.  I thought we would put the B hadron mass assumption.  This is the mass of the jet.  Are these two things compatible by construction?